### PR TITLE
test(modal): Add modal snapshots

### PR DESCRIPTION
**Pull request by anicholls**
Partially addresses #683

### CODE REVIEW

File: modules/modal/react/stories/stories_VisualTesting.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      })}
+      columnProps={[{label: '', props: {className: ''}}]}
+    >
+      {({heading, ...props}) => (
+        <Modal
+          {...props}
+          heading={heading}
+          open={true}
+          handleClose={noop}
+          style={{position: 'relative', width: 'auto', height: 'auto', padding: 32}}
```
Comment (anicholls) [inline modules/modal/react/stories/stories_VisualTesting.tsx:50]: Note: This isn't ideal - I have to apply these styles to get the overlays to show up nicely all on one page. Open to other ideas, but this seemed like the only way without creating a story for each.

Comment (alexandrzavalii) [reply modules/modal/react/stories/stories_VisualTesting.tsx:50]: Looks like default for `width` and `height` is `auto`

File: modules/modal/react/stories/stories_VisualTesting.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        >
+          <p>Are you sure you'd like to delete the item titled 'My Item'?</p>
+          <Button style={{marginRight: '16px'}} variant={Button.Variant.Delete}>
```
Comment (alexandrzavalii) [inline modules/modal/react/stories/stories_VisualTesting.tsx:53]: Maybe for the sake of consistency just always pass a number for default units?  ```marginRight: 16 ```

File: modules/modal/react/stories/stories_VisualTesting.tsx (hunk 1 above)
Comment (anicholls) [reply modules/modal/react/stories/stories_VisualTesting.tsx:50]: This is getting applied to the wrapper overlay, not the modal itself. The defaults for these are `100vw/h`

**Comment by NicholasBoll** [comment]
For whatever reason, the Chrome and Firefox snapshots are appearing empty: https://www.chromatic.com/snapshot?appId=5d854c26ba934e0020f5e98a&id=5ed7d0ea8e6105002284bdd4

I think that's because of the animation. On IE it is showing up.

**Pull request merged**